pkgs/by-name/br/brickstore/package.nix

```diff
@@ -0,0 +1,93 @@
+{
+  lib,
+  stdenv,
+  qt6,
+  libsForQt5,
+  fetchFromGitHub,
+  gst_all_1,
+  cmake,
+  libglvnd,
+  tbb,
+  ninja,
+  pkg-config,
+}:
+let
+  inherit (libsForQt5) qcoro;
+in
+stdenv.mkDerivation (finalAttrs: {
+  pname = "brickstore";
+  version = "0-unstable-2024-05-02";
+
+  src = fetchFromGitHub {
+    owner = "rgriebl";
+    repo = finalAttrs.pname;
```

Comment:
Please don't use this.

`repo = pname;` is nice for DRY but creates a binding that goes too far.

See further information about this here: https://github.com/nix-community/nixpkgs-lint/issues/21